easybuild/easyconfigs/p/PyBERT/PyBERT-1.7.9_2to3.patch

```diff
@@ -0,0 +1,415 @@
+# Python 2to3
```

Comment:
@boegel: it seems to work. There are some code that I changed by hand, not only `2to3`, although   I was not clear. The user is testing it....